app/components/Form/ProjectEmissionIntensityReportForm.tsx

```diff
@@ -24,6 +24,47 @@ interface Props {
   onSubmit: () => void;
 }
 
+// passing content suffix to fields that need nominator and denominator (example: tCO2e/GJ)
+export const createEmissionIntensityReportUiSchema = (
+  emissionFunctionalUnit: string,
+  productionFunctionalUnit?: string,
+  viewOnly?: boolean
+) => {
+  // setting a deep copy of the ui schema to avoid mutating the original
+  const uiSchemaCopy = JSON.parse(
+    JSON.stringify(emissionIntensityReportUiSchema)
+  );
+
+  // Example: tCO2e/GJ if we have emissionFunctionalUnit
+  const contentSuffix: JSX.Element = emissionFunctionalUnit && (
+    <b>{`${emissionFunctionalUnit}${
+      productionFunctionalUnit ? `/${productionFunctionalUnit}` : ""
+    }`}</b>
+  );
+
+  // We only show the label of this field on view mode and summary page
+  if (viewOnly) uiSchemaCopy.productionFunctionalUnit["ui:label"] = "";
+
+  uiSchemaCopy.baselineEmissionIntensity["ui:options"] = {
+    ...uiSchemaCopy.baselineEmissionIntensity["ui:options"],
+    contentSuffix,
+  };
+  uiSchemaCopy.targetEmissionIntensity["ui:options"] = {
+    ...uiSchemaCopy.targetEmissionIntensity["ui:options"],
+    contentSuffix,
+  };
+  uiSchemaCopy.postProjectEmissionIntensity["ui:options"] = {
+    ...uiSchemaCopy.postProjectEmissionIntensity["ui:options"],
+    contentSuffix,
+  };
+  uiSchemaCopy.totalLifetimeEmissionReduction["ui:options"] = {
+    ...uiSchemaCopy.totalLifetimeEmissionReduction["ui:options"],
+    contentSuffix,
```

Comment:
`totalLifetimeEmissionReduction` isn't a rate. It can just get `emissionFunctionalUnit` for a content suffix.

Reply:
Thanks for mentioning that 🙏 I missed it.